### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by saadat-ali-awan**
You will find all the details of the Bookstore design in [Zeplin](https://zeplin.io/).
- [Login to Zeplin](https://app.zeplin.io/login) (hint: you will find the credentials in the [sneak peek](./sneak_peek.md) of the project).
- Open the project *Bookstore CMS*.
- Use the information detailed in Zeplin to style the website.
- Your final design should match Zeplin's design.

### CODE REVIEW

File: src/components/NavMenu.jsx [hunk 1]
```diff
@@ -3,17 +3,17 @@ import styles from './NavMenu.module.css';
...
             className={({ isActive }) => `${styles['nav-link']} ${(isActive ? styles.selected : '')}`}
           >
             Books
           </NavLink>
         </li>
-        <li>
+        <li className={styles.listItem}>
           <NavLink
             to="/categories"
             className={({ isActive }) => `${styles['nav-link']} ${(isActive ? styles.selected : '')}`}
```
Comment (elmar8287) [inline src/components/NavMenu.jsx:19]: - [ ] Please consider that there is nothing shown on the screen when I click the check status button. Kindly fix that.

File: src/App.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ function App() {
     <div className="App">
       <Router>
         <header className="App-header">
-          <h1>Bookstore</h1>
+          <h1>Bookstore CMS</h1>
           <NavMenu />
         </header>
         <Routes>
```
Comment (elmar8287) [inline src/App.js]: - [ ] Please, consider that some points are not cover the requested styling. Here on this screenshot, I have marked some points to be fixed.

1. Try to move and align the section with the header "Bookstore CMS" and links to the right, to be on the same line with the books list 👍 
2. Please, try to insert the profile icon where I marked it, as is shown on the original design.
3. Please, consider fixing the styling of the buttons - they should be without the border and fixed in the needed color.
4. "Books" and "Categories" should be uppercase
5. Please, try to insert a divider between "Comments", "Remove" and "Edit"

>Your design
Screen Shot 2022-03-30 at 23 27 04

> The requested styling:
Screen Shot 2022-03-30 at 23 26 51

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @Saadat123456.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

- No linter issues 👍 
- Nice README file 👍 

## Required Changes ♻️

- [ ] Check the comments under the review.

## Optional suggestions

- Please, try to fix the link to the MIT.md file - it is broken.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BrianSammit — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello @Saadat123456, 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional 
- Your repo is professional 
- Well structured files 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**